src/views/Player.vue

```diff
@@ -59,6 +83,17 @@ export default defineComponent({
     });
     const isQuizLoaded = computed(() => state.questions.length > 0);
```

Comment:
I'm probably nitpicking now, but I guess we can have a computed variable which will calculate the `state.questions.length`. It's cached anyway and it seems to be used multiple times.